### ISSUE TITLE
Consider updating polling interval globally/for all tests

### ISSUE THREAD

**Issue by davidyuk** [aeternity/aepp-sdk-js#1075]
inspired by https://github.com/aeternity/aepp-sdk-js/pull/1072

**Comment by marc0olo** [comment]
@davidyuk @mradkov I updated this issue now after the merge of #1316 

we can address this if https://github.com/aeternity/aeternity/issues/3644 is clarified & implemented

**Comment by marc0olo** [comment]
with the latest version of devmode we should be able to cover this. not sure if the latest master-bundle of the node already includes this now.

maybe @nikita-fuchs or @thepiwo can confirm

**Comment by thepiwo** [comment]
@marc0olo this feature won't help for a good solution as I also discussed in chat with the question how that information will be used. As microblocks are not generated at expected_mine_rate and micro_block_cycle, it is only an lower bound between microblocks.

When we wait for longer than expected_mine_rate and micro_block_cycle between transactions then the next microblock will be instant, but then we will still run in our interval and delay the return.

When the querying of the node and generation of microblock may take a few ms in the node, we maybe wait for 2x our expected_mine_rate and micro_block_cycle.

It can be a hint of what interval to choose, but what number do we actually set? Is there a real performance bottleneck in the node when we query at a higher speed every time?

**Comment by marc0olo** [comment]
> When we wait for longer than expected_mine_rate and micro_block_cycle between transactions then the next microblock will be instant, but then we will still run in our interval and delay the return.

fair point. I think for the SDK usage in general we are fine with your changes in #1316. what do others think?

for the devmode we could definitely introduce some check and set the interval accordingly. or does anything speak against this? I mean this should hopefully be reliable.

**Comment by thepiwo** [comment]
we can detect devmode or local node e.g. and go to 50ms interval as locally it doesn't matter at all
